libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -41,6 +41,9 @@ singleton AP_Arming alias arming
 singleton AP_Arming method disarm boolean
 singleton AP_Arming method is_armed boolean
 singleton AP_Arming method arm boolean AP_Arming::Method::SCRIPTING'literal
+singleton AP_Arming method get_aux_auth_id boolean uint8_t'Null
+singleton AP_Arming method set_aux_auth_passed void uint8_t 0 255
```

Comment:
Why not expose `aux_auth_count_max` as a public member of AP_Arming so that you can use it here for bounds checking from a script? I know it's bound checked inside of AP_Arming, but the advantage of the check here (which we will be running either way) is that if its a larger ID it actually throws a meaningful error in the script that the argument was out of range, which lets people check what happened, rather then thinking it's just not working.

Also `UINT8_MAX` rather then 255 :)

Reply:
With the removal of the AUX_AUTH_STATE_UNASSIGNED, I've changed the range checking within AP_Arming to use aux_auth_count (the number of allocated authorisers).

Does this kind of dynamic range checking work?  I.e. I create a function that returns a uint8 and it will be used before each call to set_aux_auth_passed?

I thought we were trying to keep the number of bindings small because each one consumes a lot of memory..

I've changed the limits to use UINT8_MAX, thanks.

Reply:
I tried but failed to get the "ud->get_aux_auth_count()" to work for range checking the input to the other functions (it always failed) so I'll leave this suggestion out.  I think we can probably all agree it's nice to have but not a necessity.